tableauserverclient/server/request_options_base.py

```diff
@@ -0,0 +1,9 @@
+import abc
+
+
+class RequestOptionsBase(object):
```

Comment:
I think the abc and metaclass magic is unnecessary and doesn't really buy us anything but complexity to new contributors. It's kinda black arts level stuff :)